src/services/blocks/trace/Trace.ts

```diff
@@ -614,7 +614,7 @@ export class Trace {
 							field2: 'free',
 						},
 						amount: {
-							value: free,
+							value: free.toNumber(),
```

Comment:
if we need to do this we should do something like `.toString(10)`, else risk overflowing the javascript number type since these represent u128

Reply:
Yea totally agree!